controllers/app.controllers.js

```diff
@@ -22,7 +22,11 @@ exports.getEndpoints = (req, res, next) => {
 
 exports.getArticles = (req, res, next) => {
     fetchArticles().then((articles) => {
-        res.status(200).send({ articles })
+        if (articles.length === 0) {
```

Comment:
This _could_ happen but is not really an error to concern ourselves with. If you want to keep this, it should go in the model and be a Promise.reject that leads to the error handler in your app